package.json

```diff
@@ -11,7 +11,7 @@
     "lodash": "^4.17.21",
     "react": "^18.2.0",
     "react-redux": "^8.1.1",
-    "terraso-backend": "github:techmatters/terraso-backend#69ae4c9",
+    "terraso-backend": "github:techmatters/terraso-backend#a431376f71fe8537df6976cf8c2f6772e8d659af",
```

Comment:
Question: Is this a commit that is already in the backend `main` branch?
Suggestion: If that is the case then you can use just the first 7 characters fo the commit hash